src/ostorlab/serve_app/export_utils.py

```diff
@@ -0,0 +1,226 @@
+        )
+        if len(assets) == 0:
+            return None
+        assets_data = []
+        for asset in assets:
+            asset_type = (
+                asset.type.replace("_", " ").lower()
+                if "_" in asset.type
+                else asset.type
+            )
+            asset_dict = {
+                "tags": None,
+                "type": asset_type.lower(),
+            }
+            if asset.type == "android_file" or asset.type == "ios_file":
+                if asset.path is not None:
+                    file_asset_path = pathlib.Path(asset.path)
+                    if file_asset_path.exists() is False:
+                        raise ValueError(
+                            f"{asset_type.capitalize()} File {asset.path} not found."
+                        )
+                    try:
+                        archive.writestr(MOBILE_APP, file_asset_path.read_bytes())
+                    except Exception:
+                        pass
+
+            elif asset.type == "android_store":
+                asset_dict["package_name"] = asset.package_name
+                asset_dict["application_name"] = asset.application_name
+            elif asset.type == "ios_store":
+                asset_dict["package_name"] = asset.bundle_id
+                asset_dict["application_name"] = asset.application_name
+            elif asset.type == "network":
+                networks = []
+                ips = (
+                    session.query(models.IPRange)
+                    .filter(models.IPRange.network_asset_id == asset.id)
+                    .all()
+                )
+                for ip in ips:
+                    ip_network = ipaddress.ip_network(ip.host, strict=False)
+                    networks.append(
+                        f"{ip_network.network_address.exploded}/{ip_network.prefixlen}"
+                    )
+                asset_dict["networks"] = networks
+            elif asset.type == "urls":
+                urls = []
+                links = (
+                    session.query(models.Link)
+                    .filter(models.Link.urls_asset_id == asset.id)
+                    .all()
+                )
+                for link in links:
+                    urls.append(link.url)
+                asset_dict["urls"] = urls
+            else:
+                raise NotImplementedError()
+
+            assets_data.append(asset_dict)
+
+        assets_data = "\n".join([json.dumps(asset) for asset in assets_data])
+        archive.writestr(ASSET_JSON, assets_data)
+
+
+def _export_scan(scan: models.Scan, archive: zipfile.ZipFile) -> None:
+    """Export the scan details to the given archive.
+
+    Args:
+        scan: The scan object.
+        archive: The archive object.
+    """
+
+    scan_dict = {
+        "title": scan.title,
+        "created_time": scan.created_time.strftime("%Y-%m-%d %H:%M:%S"),
+        "risk_rating": _compute_risk_rating(scan_id=scan.id),
+        "status": [],
```

Comment:
plural no?

Reply:
Im adapting to the existing one in RE. it doesnt matter as long as its same in export and import !!